plugins/auth-backend-module-oidc-provider/src/authenticator.ts

```diff
@@ -85,6 +89,15 @@ export const oidcAuthenticator = createOAuthAuthenticator({
       );
     }
 
+    const timeoutMilliseconds = config.has('timeout')
```

Comment:
Could you please add a test for the new configuration? By doing that, we make sure that the test fails if someone accidentally removes the configuration support 🙏🏻

Reply:
Also added some tests. Please have a look. Thanks!